src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,22 @@
 // Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({
+  todo: {
+    completed,
+    title,
+    user,
+  },
+}) => (
+  <article className={`TodoInfo ${completed ? 'TodoInfo--completed' : ''}`}>
+    <h2 className="TodoInfo__title">{title}</h2>
+
+    <UserInfo user={user} />
```

Comment:
also, you can check existing user here (this way, your user prop in UserInfo component will be required)

```suggestion
    {user && (<UserInfo user={user} />)}
```